scripts/common.sh

```diff
@@ -58,8 +58,13 @@ EOF
 deploy() {
 	NAME=$1
 	ARGS=${@:2}
+
+	# find file path
+	CONTRACT_PATH=$(find -name $NAME.sol)
```

Comment:
On macOS, this fails with `find: illegal option -- n`, since the first argument need to be the directory to search in. Perhaps `find` works differently on Linux? But for macOS the fix is this

```suggestion
	CONTRACT_PATH=$(find . -name $NAME.sol)
```

Reply:
Ah, I am using linux so that must be the problem. I have confirmed that this solution works on linux as well though,

Reply:
Yea probably, I can do that now—thanks for verifying!